app/assets/javascripts/match.js

```diff
@@ -0,0 +1,65 @@
+// Gets the GameId number from the URL and returns it.
+function getGameId() {
```

Comment:
I could conceive of an `ApplicationController` instance created on `ready` which holds this method?